tfhe/examples/utilities/params_to_file.rs

```diff
@@ -7,8 +7,8 @@ use tfhe::core_crypto::prelude::{DynamicDistribution, TUniform, UnsignedInteger}
 use tfhe::keycache::NamedParam;
 use tfhe::shortint::parameters::classic::compact_pk::ALL_PARAMETER_VEC_COMPACT_PK;
 use tfhe::shortint::parameters::classic::gaussian::ALL_PARAMETER_VEC_GAUSSIAN;
-use tfhe::shortint::parameters::multi_bit::ALL_MULTI_BIT_PARAMETER_VEC;
-use tfhe::shortint::parameters::p_fail_2_minus_64::ks_pbs_gpu::*;
+use tfhe::shortint::parameters::multi_bit::gaussian::p_fail_2_minus_64::ks_pbs_gpu::*;
+use tfhe::shortint::parameters::multi_bit::gaussian::ALL_MULTI_BIT_PARAMETER_VEC;
 use tfhe::shortint::parameters::{
     ShortintParameterSet, PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64,
 };
```

Comment:
should add the GPU parameters for TUniform as well to make sure they are tested during the security check with the lattice estimator